<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KitConfigForm</class>
 <widget class="QDialog" name="KitConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>518</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0,2,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="nameKitHLayout" stretch="0,2,0">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="nameKitLabel">
       <property name="text">
        <string>Kit name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameKitEdit"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="mt4PathHLayout" stretch="0,0,0">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="mt4PathLabel">
       <property name="text">
        <string>Path to MT4 :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mt4PathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="mt4PathButton">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="serverHLayout" stretch="0,2,1">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="serverLabel">
       <property name="text">
        <string>Server of the terminal:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serverCBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="runTerminalButton">
       <property name="text">
        <string>Run MT4 with assistant</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nnTrainParamHLayout" stretch="1,3,1,2">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="trainingStrategyLabel">
       <property name="text">
        <string>Training strategy:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="trainingStrategyCBox"/>
     </item>
     <item>
      <widget class="QLabel" name="dataAllocationLabel">
       <property name="text">
        <string>Data allocation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataAllocationLEdit">
       <property name="toolTip">
        <string>Training_ratio + Generalization_ratio + Testing_ratio = 100%</string>
       </property>
       <property name="inputMask">
        <string>00%, 00%, 00%;_</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nnHLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="layerCountVLayout_5">
       <item>
        <widget class="QLabel" name="layerCountLabel">
         <property name="text">
          <string>Layers count:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="layerCountLEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layersSizeVLayout">
       <item>
        <widget class="QLabel" name="layersSizeLabel">
         <property name="text">
          <string>Layers size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="layersSizeLEdit">
         <property name="toolTip">
          <string>Sample: 
Layers count = 3, 
Layers size = &quot;6, 9, 3&quot;.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="selectSymbolHLayout" stretch="0,1,0,0,1,0">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QLabel" name="symbolLabel">
       <property name="text">
        <string>Symbol :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="symbolCBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="periodLabel">
       <property name="text">
        <string>Period :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="periodCBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addSymbolHLayout" stretch="2,2,1">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QRadioButton" name="inputRButton">
       <property name="text">
        <string>Input Layer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outputRButton">
       <property name="text">
        <string>Output Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addSymbolButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="Market-Analysis.qrc">
         <normaloff>:/img/plus_64.png</normaloff>:/img/plus_64.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputOutputHLayout" stretch="2,2,1">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QListWidget" name="inputListWidget"/>
     </item>
     <item>
      <widget class="QListWidget" name="outputListWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonsVLayout">
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
         <property name="icon">
          <iconset resource="Market-Analysis.qrc">
           <normaloff>:/img/delete_64.png</normaloff>:/img/delete_64.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="upButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Market-Analysis.qrc">
           <normaloff>:/img/up_64.png</normaloff>:/img/up_64.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="downButton">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="Market-Analysis.qrc">
           <normaloff>:/img/down_64.png</normaloff>:/img/down_64.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="radioHLayout" stretch="1,1,1">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <widget class="QCheckBox" name="recurrentNNChBox">
       <property name="text">
        <string>Recurrent input layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="readVolumeChBox">
       <property name="text">
        <string>Read volume data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nnInOutHLayout">
     <property name="leftMargin">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>3</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="depthInVLayout">
       <item>
        <widget class="QLabel" name="depthInLabel">
         <property name="text">
          <string>Depth input history:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="depthInLEdit">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="inputCountVLayout">
       <item>
        <widget class="QLabel" name="inputCountLabel">
         <property name="text">
          <string>Input layer size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputCountLEdit">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="depthOutVLayout">
       <item>
        <widget class="QLabel" name="depthOutLabel">
         <property name="text">
          <string>Depth prediction:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="depthOutLEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="outputCountVLayout">
       <item>
        <widget class="QLabel" name="outputCountLabel">
         <property name="text">
          <string>Output layer size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outputCountLEdit">
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Market-Analysis.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KitConfigForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KitConfigForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
